hnn_core/gui/gui.py

```diff
@@ -943,7 +943,8 @@ def on_upload_change(change, params, tstop, dt, log_out, simulation_data,
         with log_out:
             print(
                 "Same param. No reloading."
-                "To force reloading, hit \"clear uploaded parameters\" button")
+                "To force reloading, hit \"clear uploaded parameters\" button",
```

Comment:
see: https://peps.python.org/pep-0008/#string-quotes

"When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability."

Just avoid backslashes when you can :)